env.yaml

```diff
@@ -2,19 +2,21 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - python >= 3.9
-  - pandas
+  - python
```

Comment:
What python version are we running under now ? Was there a specific requirement to drop 3.10 or higher ?

Reply:
it is 3.10.13, it is solved under the env building, but seems that 3.10.X is the minimal version now